R/stylesheet.R

```diff
@@ -0,0 +1,31 @@
+is_stylesheet <- function(stylesheet) {
+  inherits(stylesheet, "xml_document") && 
+    length(xml2::xml_name(stylesheet)) == 1L    &&
+    xml2::xml_name(stylesheet) == "stylesheet"
```

Comment:
so, out-of-scope, but now I wonder how one can actually ensure something is a valid stylesheet.

Reply:
I would imagine that it would involve an XML schema and `xml2::xml_validate()`, but this is still something that I need to expand my knowledge on.

Reply:
https://github.com/ropensci/tinkr/issues/35